opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -95,14 +95,141 @@ message InstrumentationLibraryMetrics {
-//     aggregation temporality).
-//   - the instantaneous time of the event.
+// - Attributes includes key-value pairs associated with the data point
+// - TimeUnixNano is required, set to the end time of the aggregation
+// - StartTimeUnixNano is optional, but strongly encouraged for DataPoints
+//   having an AggregationTemporality field, as discussed below.
+//
+// Both TimeUnixNano and StartTimeUnixNano values are expressed as
+// UNIX Epoch time in nanoseconds since 00:00:00 UTC on 1 January 1970.
+//
+// # TimeUnixNano
+//
+// This field is required, having consistent interpretation across
+// DataPoint types.  TimeUnixNano is the moment corresponding to when
+// the data point's aggregate value was captured.
+//
+// Data points with/ the 0 value for TimeUnixNano SHOULD be rejected
+// by consumers.
+//
+// # StartTimeUnixNano
+//
+// StartTimeUnixNano in general allows detecting when a sequence of
+// observations is unbroken.  StartTimeUnixNano is set to the
+// beginning of an unbroken sequence of observations (all point kinds
+// except with delta aggregation temporality) or the end of an
+// unbroken sequence of observations (point kinds with delta
+// temporality).
+//
+// This field indicates to consumers the start time for points with
+// cumulative and delta AggregationTemporality, and it should be
+// included whenever possible to support correct rate calculation.
+// Although it may be omitted when the start time is truly unknown,
+// setting StartTimeUnixNano is strongly encouraged.
+//
+// There are two reasons that streams with AggregationTemporality are
+// reset: (1) a process restarts, and (2) the process "forgets" the stream
+// and then re-initializes it.  StartTimeUnixNano is critical for
+// calculating correct rates for Monotonic Sum, Histogram, and Summary
+// data points in both of these cases.
+//
+// Another use for this information which applies to all point kinds
+// is to prevent overlap when multiple writers accidentally write to a
+// single stream.  StartTimeUnixNano MAY be used for overlap detection
+// for all point kinds.
+//
+// ## StartTimeUnixNano: Sums and Histograms
+//
+// Since these two point kinds have AggregationTemporality, the use of
+// StartTimeUnixNano is strongly recommended to ensure correct rate
+// calculation and prevent overlap.
+//
+// ## StartTimeUnixNano: Gauges
+//
+// The use of StartTimeUnixNano is optional for Gauge points.  This is
+// recommended in cases where there is a risk of mis-interpretion caused
+// by overlapping points (by a violation of the single-writer principle).
+//
+// When used, StartTimeUnixNano should be set to the start time of the
+// process or the timestamp when the Gauge was first set.
+//
+// ## StartTimeUnixNano: Summaries
+//
+// Summary points are treated the same as for cumulative aggregation
+// temporality, by convention.  StartTimeUnixNano should be set to the
+// starting timestamp in an unbroken sequence of observations.
```

Comment:
May worth explaining at least to me this :) We don't produce this and will only get them from Prometheus, so probably what prometheus gives us that's what we propagate.

Reply:
I want to be able to say that a Histogram with zero buckets and a Summary with zero quantiles should have the same semantics _in practice_, which means that StartTime should be set as for Cumulative aggregation temporality, because Prometheus does.
For a stronger interpretation of this, see #266.